packages/inference/test/tapes.json

```diff
@@ -156,13 +156,15 @@
       "body": "{\"inputs\":{\"question\":\"What is the capital of France?\",\"context\":\"The capital of France is Paris.\"},\"options\":{}}"
     },
     "response": {
-      "body": "{\"score\":0.9703431725502014,\"start\":25,\"end\":30,\"answer\":\"Paris\"}",
-      "status": 200,
-      "statusText": "OK",
+      "body": "{\"error\":\"overloaded\"}",
+      "status": 503,
+      "statusText": "Service Unavailable",
       "headers": {
         "access-control-allow-credentials": "true",
         "connection": "keep-alive",
         "content-type": "application/json",
+        "server": "uvicorn",
+        "transfer-encoding": "chunked",
```

Comment:
Let's not add 5XX to tapes.

You can do `it.only` to only record the new test

Reply:
this is `questionAnswering` test failing since this model https://huggingface.co/deepset/roberta-base-squad2 is overloaded, I can checkout the old tapes.json and add the it.only

Reply:
I've rolled back tapes.json, for now since we need the api-inference to change the params, to make tests here pass again